pavement.py

```diff
@@ -22,7 +22,7 @@
 
 master_app = 'runestone'
 serving_dir = "./build/engr101umich"
-dynamic_pages = False
+dynamic_pages = True
```

Comment:
For the `lobster-preview` branch, `dynamic_pages` should be `False` (we switch it to `True` in the `master` branch)